frontend/components/commons/header/filters/FiltersList.vue

```diff
@@ -59,6 +59,12 @@
               :filter="filter"
               @apply="onApply"
             />
+            <FilterUncoveredByRules
+              v-else-if="filter.type === 'checkbox' && showUncoveredByRulesFilter"
```

Comment:
Do we need the `filter.type === "checkbox"` condition?

Reply:
we can remove it